src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+  TODO
+  - Look for a better icon here https://fontawesome.com/v5/search
+  - Set right title, subtitle, explanation
+-->
+<app-dashboard-widget
+  icon="birthday-cake"
+  theme="child"
+  title="Birthday"
+  subtitle="Upcoming Birthdays"
+  explanation="Temporary explanation"
+>
+  <app-widget-content>
+    <div class="mat-elevation-z8">
+      <table mat-table [dataSource]="dataSource" matSort>
+        <ng-container matColumnDef="entityId">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header>Child Name</th>
+          <td mat-cell *matCellDef="let children">{{ children.entityId}} </td>
+        </ng-container>
+
+        <ng-container matColumnDef="Birthdays">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header>Birthday Date</th>
+          <td mat-cell *matCellDef="let children">{{children.Birthday}} </td>
+        </ng-container>
+
+        <ng-container matColumnDef="age">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header>After Birthday Age</th>
+          <td mat-cell *matCellDef="let children">{{children.age}}</td>
+        </ng-container>
+
+    
+<!--
+  TODO
+  - replace this by a table similar to `AttendanceWeekDashboard` (for more info see https://v11.material.angular.io/components/table/overview)
+  - display the child using the `ChildBlockComponent`
+  - display the birthday
+  - display the age the child will have after its birthday
+-->
+      <app-child-block
+       *ngFor="let child of children"
+       [entity]="child"
+      ></app-child-block>
+      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
```

Comment:
```suggestion
      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
      <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
```
See the similar code in the [NotesDashboardComponent](https://github.com/Aam-Digital/ndb-core/blob/master/src/app/child-dev-project/notes/dashboard-widgets/notes-dashboard/notes-dashboard.component.html#L45)